src/src/WebServer/Markup_Forms.cpp

```diff
@@ -32,13 +32,7 @@ void addFormSeparator(int clspan)
 void addFormNote(const String& text, const String& id)
 {
   addRowLabel_tr_id("", id);
-  String html;
-
-  html.reserve(40 + text.length());
-  html += F("<div class='note'>Note: ");
```

Comment:
Is that Note: prefix no longer part of a note? I find it quite useful.

Reply:
Was not my intention to remove it.
This is the reason why I love it when others also have a look at the code 👍